storybook/CMakeLists.txt

```diff
@@ -17,11 +17,11 @@ endif()
 
 find_package(
   Qt5
-  COMPONENTS Core Quick QuickControls2 Test
+  COMPONENTS Core Quick QuickControls2 Test QuickTest Qml
   REQUIRED)
 
 file(GLOB_RECURSE QML_FILES "stubs/*.qml" "mocks/*.qml" "pages/*.qml"
-    "src/*.qml" "src/qmldir" "../ui/StatusQ/*.qml" "../ui/app/*.qml")
+    "src/*.qml" "src/qmldir" "qmlTests/" "../ui/StatusQ/*.qml" "../ui/app/*.qml")
```

Comment:
it's globbed incorrectly therefore test files won't appear in qtcreator, it should be e.g. `qmlTests/*.qml`. However test sources shouldn't go along with the rest sources (they will be visible under `Storybook` in qtcreator). So maybe it's good moment to split it into 3 parts: core (statusq/app), storybook-specific and unittests-specific. Wdyt?

Reply:
Agree. I'll update the commit. Will take the chance and add ../ui/imports/*.qml as well for convenience.